tests/service/server_sanic_test.py

```diff
@@ -8,10 +8,20 @@
 
 class TestApp(AllenNlpTestCase):
 
-    app = make_app()
-    app.servables = ServableCollection.default()
-    app.testing = True
-    client = app.test_client
+    client = None
+
+    def setUp(self):
+        super(TestApp, self).setUp()
+        if self.client is None:
+            app = make_app()
+            app.servables = ServableCollection.default()
+            app.testing = True
+            self.client = app.test_client
+
+    def tearDown(self):
+        super(TestApp, self).tearDown()
+        os.remove('access.log')
+        os.remove('error.log')
```

Comment:
do you think it's too defensive to check if they exist? as it stands this will probably crash if they don't get created for some reason